containerjs-api-utility is a required dependency

containerjs-api-utility is a required dependency and should be published as other packages fail to install:

```shell
λ npm install -g containerjs-api-electron
npm ERR! No compatible version found: containerjs-api-utility@^0.0.4
npm ERR! Valid install targets:
npm ERR! 0.0.3, 0.0.2
```

--- a/packages/api-utility/package.json
+++ b/packages/api-utility/package.json
@@ -2,7 +2,6 @@
   "name": "containerjs-api-utility",
   "version": "0.0.4",
   "description": "Utility classes and functions for ContainerJS",
-  "private": true,
   "main": "build/es/index.js",
   "typings": "build/es/index",
   "repository": {

